app/assets/javascripts/stores/message_store.js.coffee

```diff
@@ -0,0 +1,55 @@
+window.messageStore = Reflux.createStore
+  init: ->
+    @listenToMany(userActions)
+    @listenTo remoteActions.message, 'onReceiveMessage'
+    @state = @getInitialState()
```

Comment:
That won't work well because @getInitialState will be triggered again after @init. http://stackoverflow.com/questions/29308672/reflux-trigger-wont-work-without-a-delay-in-init